Web/src/components/room/PreparedComponent.tsx

```diff
@@ -0,0 +1,147 @@
+import React, { lazy, useState } from 'react';
+import {
+  Users, UserX, UserCheck, Icon,
+} from 'react-feather';
+import BigButton from 'components/common/BigButton';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components/macro';
+import Modal from 'components/common/Modal';
+import { ListItemContainer, ListItem } from 'components/List';
+import Game from './Games';
+import { ResultMap } from './comm/Comm';
+
+const BalloonShakeInstructions = lazy(() => import('components/games/BalloonShake/BalloonShakeInstructions'));
+const MentalSumsInstructions = lazy(() => import('components/games/MentalSums/MentalSumsInstructions'));
+const PandaSequenceInstructions = lazy(() => import('components/games/PandaSequence/PandaSequenceInstructions'));
+
+export interface PreparedComponentProps {
+  isReady: boolean;
+  onReadyClick: () => void;
+  game: Game;
+  allMetas: ResultMap | null;
+}
+
+interface PlayerInfoModalProps {
+  isVisible: boolean;
+  allMetas: ResultMap;
+  closeAction: () => void;
+}
+
+const InverseButton = styled(BigButton)`
+  background: white;
+  color: var(--purple);
+`;
+
+const WhiteLink = styled(Link)`
+  color: white;
+`;
+
+const UsersIcon = styled(Users as React.FC<React.ComponentProps<Icon>>)`
+  stroke-width: 3;
+`;
+
+const UserCheckIcon = styled(UserCheck as React.FC<React.ComponentProps<Icon>>)`
+  stroke-width: 3;
+  color: var(--green);
+`;
+
+const UserXIcon = styled(UserX as React.FC<React.ComponentProps<Icon>>)`
+  stroke-width: 3;
+  color: var(--red);
+`;
+
+const NameSpan = styled.span`
+  width: calc(100% - 2em);
+  text-overflow: ellipsis;
+  overflow: hidden;
+`;
+
+const PlayerListContainer = styled(ListItemContainer)`
+  width: 100%;
```

Comment:
When we have to many players connected, the list will flex it's way out of the screen hahaha so let's add this 

```suggestion
  width: 100%;
  max-height: 50vh;
  overflow-y: auto;
```